### ISSUE TITLE
Упростить Metronome::paCallback

### ISSUE THREAD

**Issue by mofr** [cruvar/gui_metronome#2]
Создать приватный метод в классе `Metronome`, который будет выполнять всю работу текущего `paCallback`. Нужно для упрощения кода, чтобы все время не писать `metronome->`

**Comment by cruvar** [comment]
@mofr это относится к указателям на методы чтоль?

**Comment by mofr** [comment]
Нет, указатели на методы не нужны, в `paCallback` уже передается указатель на метроном, достаточно там вызывать нечто вроде `metronome->fillBuffer(...)`. `paCallback` нужна только как прокси-функция между мирами C и C++.
Это не фундаментальное изменение, но мне кажется что код станет почище.
